src/test/java/bio/terra/janitor/db/JanitorDaoTest.java

```diff
@@ -39,13 +50,62 @@ public void createTrackedResource() throws Exception {
     CloudResourceUid cloudResourceUid = newGoogleProjectResourceUid();
     janitorDao.createResource(cloudResourceUid, DEFAULT_LABELS, CREATION, EXPIRATION);
```

Comment:
should the created TrackedResourceId also be asserted against?

Reply:
Done